app/src/renderer/scripts/time.js

```diff
@@ -2,13 +2,13 @@
 
 const getUnbondingTime = ({ height, tx }, unbondingDelegations) => {
   const { type, value } = tx.value && tx.value.msg && tx.value.msg[0]
-  if (type === `cosmos-sdk/Undelegate`) {
-    const unbondingDelegation = unbondingDelegations[value.validator_addr]
+  if (type === `cosmos-sdk/MsgUndelegate`) {
+    const validatorUnbondingDelegation = unbondingDelegations[value.validator_address]
```

Comment:
is `validatorUnbondingDelegation` being used?

the naming is a bit confusing too. 

`validatorUnbondingDelegation = unbondingDelegations[value.validator_address]`
`unbondingDelegation = validatorUnbondingDelegation.find()`